packages/components/combobox/src/combobox.ts

```diff
@@ -503,6 +517,30 @@ export class Combobox<T = any, U = T> extends FormControlMixin(ScopedElementsMix
       }
     } else if (!this.wrapper?.matches(':popover-open') && ['ArrowDown', 'ArrowUp'].includes(event.key)) {
       this.wrapper?.showPopover();
+    } else if (['ArrowLeft', 'ArrowRight'].includes(event.key) && this.input.selectionStart === 0) {
```

Comment:
In some cases I can get two tags focused at once (with arrow keys):

<img width="1352" alt="image" src="https://github.com/user-attachments/assets/c1d893d1-252a-4c9c-8450-0cf5cf969357">